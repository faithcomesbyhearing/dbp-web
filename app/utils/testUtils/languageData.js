export const languages = [
	{
		translations: {
			472: 'azərbaycan dili',
			943: 'azerbejdžanski',
			1232: 'азербайджанийн',
			1431: 'ázerbájdžánština',
			1708: 'ཨ་ཛར་བྷའི་ཇཱན་ཁ',
			1867: 'azerbajĝana',
			1887: 'azerbaijangbe',
			1910: 'ترکی آذربایجانی',
			1928: 'azeri',
			2315: 'Azerbaijanci',
			2326: 'אזרית',
			2472: 'aserska',
			2827: 'aserbajdsjaniskisut',
			2971: 'اَزَربیجانی',
			3154: 'Inyazeribayijani',
			3612: 'azerbaidžāņu',
			3853: 'Aserbaidschanesch',
			4003: 'Ażerbajġani',
			5043: 'ադրբեջաներեն',
			5045: 'ଆଜେରବାଇଜାନି',
			5047: 'Azerice',
			5048: 'აზერბაიჯანული',
			5277: 'Afaan Azerbaijani',
			5298: 'тӕтӕйраг',
			5580: 'azerski',
			5650: 'أذربائجاني',
			5676: 'Azerbaiyano Simi',
			5778: 'aserbeidschanic',
			5779: 'azeră',
			5812: 'азербайджанский',
			6076: 'азербејџански',
			6242: 'azerbajdžančina',
			6259: 'Azerbaijan',
			6413: 'azéri',
			6414: 'Azeri',
			17045: 'Azeri',
			6415: 'Aserbaidschanisch',
			6799: 'አዜርባይጃንኛ',
			6891: 'lea fakaʻasapaisani',
			7019: 'azerbaýjança',
			7078: 'ئەزەرىچە',
			7087: 'азербайджанська',
			7179: 'ozarbayjon',
			7742: 'אַזערביידזשאַניש',
			7782: 'Èdè Azerbaijani',
			7861: '阿塞拜疆文',
			7947: 'aserbajdsjansk',
		},
		iso: 'aze',
		name: 'Azerbaijani, North',
		filesets: 6,
		alt_names: [
			'azərbaycan dili',
			'azerbejdžanski',
			'азербайджанийн',
			'ázerbájdžánština',
			'ཨ་ཛར་བྷའི་ཇཱན་ཁ',
			'azerbajĝana',
			'azerbaijangbe',
			'ترکی آذربایجانی',
			'azeri',
			'Azerbaijanci',
			'אזרית',
			'aserska',
			'aserbajdsjaniskisut',
			'اَزَربیجانی',
			'Inyazeribayijani',
			'azerbaidžāņu',
			'Aserbaidschanesch',
			'Ażerbajġani',
			'ադրբեջաներեն',
			'ଆଜେରବାଇଜାନି',
			'Azerice',
			'აზერბაიჯანული',
			'Afaan Azerbaijani',
			'тӕтӕйраг',
			'azerski',
			'أذربائجاني',
			'Azerbaiyano Simi',
			'aserbeidschanic',
			'azeră',
			'азербайджанский',
			'азербејџански',
			'azerbajdžančina',
			'Azerbaijan',
			'azéri',
			'Azeri',
			'Aserbaidschanisch',
			'አዜርባይጃንኛ',
			'lea fakaʻasapaisani',
			'azerbaýjança',
			'ئەزەرىچە',
			'азербайджанська',
			'ozarbayjon',
			'אַזערביידזשאַניש',
			'Èdè Azerbaijani',
			'阿塞拜疆文',
			'aserbajdsjansk',
		],
		autonym: 'AZƏRBAYCAN',
		glotto_id: 'aze',
		vernacular_name: 'AZƏRBAYCAN',
		id: 472,
		englishName: 'Azerbaijani, North',
		bibles: 3,
	},
	{
		translations: {
			1: 'Aarai',
			6414: 'Aari',
			17045: 'Aari',
			8012: 'Shankillinya',
		},
		iso: 'aiw',
		name: 'Aari',
		filesets: 2,
		alt_names: ['Aarai', 'Aari', 'Shankillinya'],
		autonym: 'Aari',
		glotto_id: 'aari1239',
		vernacular_name: 'Aari',
		id: 1,
		englishName: 'Aari',
		bibles: 2,
	},
	{
		translations: {
			9: 'Abau',
			6414: 'Abau',
			17045: 'Abau',
			8012: 'Green River',
		},
		iso: 'aau',
		name: 'Abau',
		filesets: 3,
		alt_names: ['Abau', 'Green River'],
		autonym: 'Abau',
		glotto_id: 'abau1245',
		vernacular_name: 'Abau',
		id: 9,
		englishName: 'Abau',
		bibles: 1,
	},
	{
		translations: { 13: 'Abiji', 6414: 'Abidji', 17045: 'Abidji' },
		iso: 'abi',
		name: 'Abiji',
		filesets: 4,
		alt_names: ['Abiji', 'Abidji'],
		autonym: 'Abiji',
		glotto_id: 'abid1235',
		vernacular_name: 'Abiji',
		id: 13,
		englishName: 'Abiji',
		bibles: 2,
	},
	{
		translations: { 24: 'A Nden', 6414: 'Abun', 17045: 'Abun', 8012: 'Yimbun' },
		iso: 'kgr',
		name: 'Abun',
		filesets: 1,
		alt_names: ['A Nden', 'Abun', 'Yimbun'],
		autonym: 'Abun',
		glotto_id: 'abun1252',
		vernacular_name: 'Abun',
		id: 24,
		englishName: 'Abun',
		bibles: 2,
	},
	{
		translations: {
			35: 'Aceh',
			219: 'አቻይንኛ',
			312: 'الأتشينيزية',
			472: 'akin',
			748: 'অ্যাচাইনিজ',
			959: 'achineg',
			1003: 'ачински',
			1405: 'achinese',
			1431: 'acehština',
			1472: 'achinesisk',
			1688: 'Atjehs',
			1871: 'atšehi',
			1910: 'آچئی',
			1928: 'atšeh',
			2208: 'અચીની',
			2326: 'אכינזית',
			2355: 'अचाइनीस',
			2438: 'achinéz',
			2472: 'akkíska',
			2534: 'Aceh',
			2595: 'accinese',
			2971: 'اَچَےنیٖز',
			3276: '아체어',
			3589: 'ແອັກຊີເນັສ',
			3612: 'ačinu',
			3734: 'ačinezų',
			3853: 'Aceh-Sprooch',
			3879: 'ачешки',
			3964: 'അചിനീസ്',
			4003: 'Aċiniż',
			4100: 'अचीनी',
			4409: 'Αχινίζ',
			4754: 'अचाइनिज',
			4934: 'acehgiella',
			5024: 'achinesisk',
			5043: 'աչեհերեն',
			5045: 'ଆଚାଇନୀଜ୍',
			5047: 'Achinese',
			5048: 'აჩეხური',
			5049: 'ಅಛಿನೀಸ್',
			5073: 'アチェ語',
			5391: 'ਅਚੀਨੀ',
			5580: 'aceh',
			5598: 'achém',
			5778: 'aceh',
			5779: 'aceh',
			5812: 'ачехский',
			6014: 'Basa Acèh',
			6241: 'ačejščina',
			6242: 'acehčina',
			6411: 'acehnés',
			6412: 'atjeh',
			6413: 'aceh',
			6414: 'Acehnese',
			17045: 'Acehnese',
			6415: 'Aceh-Sprache',
			6484: 'acehnesiska',
			6589: 'ஆச்சினீஸ்',
			6696: 'ఆఖినీస్',
			6762: 'อาเจะห์',
			6891: 'lea fakaʻatisē',
			7078: 'ئاتجېچە',
			7087: 'ачехська',
			7224: 'Tiếng Achinese',
			7395: 'Acehneg',
			7406: 'Atjeesk',
			7947: 'achinesisk',
			8012: 'Achinese',
		},
		iso: 'ace',
		name: 'Aceh',
		filesets: 4,
		alt_names: [
			'Aceh',
			'አቻይንኛ',
			'الأتشينيزية',
			'akin',
			'অ্যাচাইনিজ',
			'achineg',
			'ачински',
			'achinese',
			'acehština',
			'achinesisk',
			'Atjehs',
			'atšehi',
			'آچئی',
			'atšeh',
			'અચીની',
			'אכינזית',
			'अचाइनीस',
			'achinéz',
			'akkíska',
			'accinese',
			'اَچَےنیٖز',
			'아체어',
			'ແອັກຊີເນັສ',
			'ačinu',
			'ačinezų',
			'Aceh-Sprooch',
			'ачешки',
			'അചിനീസ്',
			'Aċiniż',
			'अचीनी',
			'Αχινίζ',
			'अचाइनिज',
			'acehgiella',
			'աչեհերեն',
			'ଆଚାଇନୀଜ୍',
			'Achinese',
			'აჩეხური',
			'ಅಛಿನೀಸ್',
			'アチェ語',
			'ਅਚੀਨੀ',
			'aceh',
			'achém',
			'ачехский',
			'Basa Acèh',
			'ačejščina',
			'acehčina',
			'acehnés',
			'atjeh',
			'Acehnese',
			'Aceh-Sprache',
			'acehnesiska',
			'ஆச்சினீஸ்',
			'ఆఖినీస్',
			'อาเจะห์',
			'lea fakaʻatisē',
			'ئاتجېچە',
			'ачехська',
			'Tiếng Achinese',
			'Acehneg',
			'Atjeesk',
		],
		autonym: 'Aceh',
		glotto_id: 'achi1257',
		vernacular_name: 'Aceh',
		id: 35,
		englishName: 'Aceh',
		bibles: 1,
	},
	{
		translations: {
			30: 'Ajagua',
			6414: 'Achagua',
			17045: 'Achagua',
			8012: 'Xagua',
		},
		iso: 'aca',
		name: 'Achagua',
		filesets: 3,
		alt_names: ['Ajagua', 'Achagua', 'Xagua'],
		autonym: 'Achagua',
		glotto_id: 'acha1250',
		vernacular_name: 'Achagua',
		id: 30,
		englishName: 'Achagua',
		bibles: 1,
	},
	{
		translations: {
			40: 'Acholi',
			68: 'Akoli',
			219: 'አኮሊኛ',
			312: 'الأكولية',
			472: 'akoli',
			664: 'Acholiera',
			748: 'আকোলি',
			959: 'acoli',
			1003: 'аколи',
			1405: 'acoli',
			1431: 'akolština',
			1472: 'acoli',
			1688: 'Akoli',
			1871: 'atšoli',
			1910: 'آچولیایی',
			1928: 'atšoli',
			1994: 'acoli',
			2208: 'એકોલી',
			2326: 'אקולי',
			2355: 'अकोली',
			2438: 'akoli',
			2472: 'acoli',
			2534: 'Acoli',
			2595: 'acioli',
			2971: 'اَکولی',
			3276: '아콜리어',
			3589: 'ອາໂຄລີ',
			3612: 'ačolu',
			3734: 'akolių',
			3853: 'Acholi-Sprooch',
			3879: 'аколи',
			3964: 'അകോലി',
			4003: 'Akoli',
			4100: 'अकोली',
			4409: 'Ακολί',
			4517: 'Akoli',
			4754: 'अकोली',
			5024: 'acoli',
			5043: 'ակոլերեն',
			5045: 'ଆକୋଲି',
			5047: 'Acoli',
			5048: 'აჩოლი',
			5049: 'ಅಕೋಲಿ',
			5073: 'アチョリ語',
			5391: 'ਅਕੋਲੀ',
			5580: 'aczoli',
			5598: 'acoli',
			5778: 'acoli',
			5779: 'acoli',
			5812: 'ачоли',
			6014: 'Acoli',
			6241: 'ačolijščina',
			6242: 'ačoli',
			6411: 'acoli',
			6412: 'acoli',
			6413: 'acoli',
			6414: 'Acoli',
			17045: 'Acoli',
			6415: 'Acholi-Sprache',
			6478: 'Kiakoli',
			6484: 'acholi',
			6589: 'அகோலி',
			6696: 'అకోలి',
			6762: 'อาโคลิ',
			6891: 'lea fakaʻakoli',
			7078: 'ئاچولىچە',
			7087: 'ачолі',
			7147: 'اکولی',
			7224: 'Tiếng Acoli',
			7395: 'Acoli',
			7406: 'Akoli',
			7883: 'Isi-Acoli',
			7947: 'acoli',
			8012: 'Shuli',
		},
		iso: 'ach',
		name: 'Acholi',
		filesets: 4,
		alt_names: [
			'Acholi',
			'Akoli',
			'አኮሊኛ',
			'الأكولية',
			'akoli',
			'Acholiera',
			'আকোলি',
			'acoli',
			'аколи',
			'akolština',
			'atšoli',
			'آچولیایی',
			'એકોલી',
			'אקולי',
			'अकोली',
			'Acoli',
			'acioli',
			'اَکولی',
			'아콜리어',
			'ອາໂຄລີ',
			'ačolu',
			'akolių',
			'Acholi-Sprooch',
			'അകോലി',
			'Ακολί',
			'ակոլերեն',
			'ଆକୋଲି',
			'აჩოლი',
			'ಅಕೋಲಿ',
			'アチョリ語',
			'ਅਕੋਲੀ',
			'aczoli',
			'ачоли',
			'ačolijščina',
			'ačoli',
			'Acholi-Sprache',
			'Kiakoli',
			'acholi',
			'அகோலி',
			'అకోలి',
			'อาโคลิ',
			'lea fakaʻakoli',
			'ئاچولىچە',
			'ачолі',
			'اکولی',
			'Tiếng Acoli',
			'Isi-Acoli',
			'Shuli',
		],
		autonym: 'Acholi',
		glotto_id: 'acol1236',
		vernacular_name: 'Acholi',
		id: 40,
		englishName: 'Acholi',
		bibles: 1,
	},
	{
		translations: {
			39: 'Achuar-Shiwiar',
			6414: 'Achuar-Shiwiar',
			17045: 'Achuar-Shiwiar',
			8012: 'Maina',
		},
		iso: 'acu',
		name: 'Achuar-Shiwiar',
		filesets: 8,
		alt_names: ['Achuar-Shiwiar', 'Maina'],
		autonym: 'Achuar-Shiwiar',
		glotto_id: 'achu1248',
		vernacular_name: 'Achuar-Shiwiar',
		id: 39,
		englishName: 'Achuar-Shiwiar',
		bibles: 1,
	},
	{
		translations: {
			33: 'Ache-Guayaki',
			6414: 'Aché',
			17045: 'Aché',
			8012: 'Guoyagui',
		},
		iso: 'guq',
		name: 'Aché',
		filesets: 5,
		alt_names: ['Ache-Guayaki', 'Aché', 'Guoyagui'],
		autonym: 'Aché',
		glotto_id: 'ache1246',
		vernacular_name: 'Aché',
		id: 33,
		englishName: 'Aché',
		bibles: 1,
	},
	{
		translations: { 34: 'Achí', 6414: 'Achi', 17045: 'Achi' },
		iso: 'acr',
		name: 'Achi',
		filesets: 15,
		alt_names: ['Achí', 'Achi'],
		autonym: 'Achí',
		glotto_id: 'achi1256',
		vernacular_name: 'Achí',
		id: 34,
		englishName: 'Achi',
		bibles: 6,
	},
	{
		translations: {
			36: 'Achí, Cubulco',
			6414: 'Achi de Cubulco',
			17045: 'Achi de Cubulco',
		},
		iso: 'acc',
		name: 'Achi de Cubulco',
		filesets: 7,
		alt_names: ['Achí, Cubulco', 'Achi de Cubulco'],
		autonym: 'Achí, Cubulco',
		glotto_id: 'achi1258',
		vernacular_name: 'Achí, Cubulco',
		id: 36,
		englishName: 'Achi de Cubulco',
		bibles: 2,
	},
	{
		translations: {
			51: 'Bedere',
			6414: 'Adele',
			17045: 'Adele',
			8012: 'Gidire',
		},
		iso: 'ade',
		name: 'Adele',
		filesets: 3,
		alt_names: ['Bedere', 'Adele', 'Gidire'],
		autonym: 'Adele',
		glotto_id: 'adel1244',
		vernacular_name: 'Adele',
		id: 51,
		englishName: 'Adele',
		bibles: 1,
	},
	{
		translations: {
			53: 'Adjukru',
			6414: 'Adioukrou',
			17045: 'Adioukrou',
			8012: 'Ajukru',
		},
		iso: 'adj',
		name: 'Adioukrou',
		filesets: 4,
		alt_names: ['Adjukru', 'Adioukrou', 'Ajukru'],
		autonym: 'Adioukrou',
		glotto_id: 'adio1239',
		vernacular_name: 'Adioukrou',
		id: 53,
		englishName: 'Adioukrou',
		bibles: 1,
	},
	{
		translations: {
			60: 'Adzera',
			6414: 'Adzera',
			17045: 'Adzera',
			8012: 'Azera',
		},
		iso: 'adz',
		name: 'Adzera',
		filesets: 3,
		alt_names: ['Adzera', 'Azera'],
		autonym: 'Adzera',
		glotto_id: 'adze1240',
		vernacular_name: 'Adzera',
		id: 60,
		englishName: 'Adzera',
		bibles: 1,
	},
	{
		translations: {
			71: 'Agarabi',
			6414: 'Agarabi',
			17045: 'Agarabi',
			8012: 'Bare',
		},
		iso: 'agd',
		name: 'Agta, Dupaninan',
		filesets: 3,
		alt_names: ['Agarabi', 'Bare'],
		autonym: 'Agarabi',
		glotto_id: 'agar1252',
		vernacular_name: 'Agarabi',
		id: 71,
		englishName: 'Agta, Dupaninan',
		bibles: 2,
	},
	{
		translations: {
			1676: 'Dupaningan Agta',
			6414: 'Dupaninan Agta',
			17045: 'Dupaninan Agta',
			8012: 'Eastern Cagayan Agta',
		},
		iso: 'duo',
		name: 'Agta, Dupaninan',
		filesets: 1,
		alt_names: ['Dupaningan Agta', 'Dupaninan Agta', 'Eastern Cagayan Agta'],
		autonym: 'Agta, Dupaninan',
		glotto_id: 'dupa1235',
		vernacular_name: 'Agta, Dupaninan',
		id: 1676,
		englishName: 'Agta, Dupaninan',
		bibles: 1,
	},
	{
		translations: {
			82: 'Palanan Agta',
			6414: 'Pahanan Agta',
			17045: 'Pahanan Agta',
			8012: 'Palanan Agta',
		},
		iso: 'apf',
		name: 'Agta, Pahanan',
		filesets: 6,
		alt_names: ['Palanan Agta', 'Pahanan Agta'],
		autonym: 'Agta, Pahanan',
		glotto_id: 'agta1234',
		vernacular_name: 'Agta, Pahanan',
		id: 82,
		englishName: 'Agta, Pahanan',
		bibles: 1,
	},
	{
		translations: {
			84: 'Aguacateco',
			6414: 'Awakateko',
			17045: 'Awakateko',
			8012: 'Aguacateco',
		},
		iso: 'agu',
		name: 'Awakateko',
		filesets: 8,
		alt_names: ['Aguacateco', 'Awakateko'],
		autonym: 'Aguacateco',
		glotto_id: 'agua1252',
		vernacular_name: 'Aguacateco',
		id: 84,
		englishName: 'Awakateko',
		bibles: 2,
	},
	{
		translations: {
			68: 'Engels',
			124: 'Borɔfo',
			219: 'እንግሊዝኛ',
			312: 'الإنجليزية',
			578: 'angilɛkan',
			664: 'ingelesa',
			728: 'англійская',
			748: 'ইংরেজি',
			959: 'saozneg',
			1003: 'английски',
			1156: 'អង់គ្លេស',
			1232: 'ингалсан',
			1322: 'а҆нглі́йскїй',
			1387: 'Sowsnek',
			1405: 'engleski',
			1431: 'angličtina',
			1472: 'engelsk',
			1688: 'Engels',
			1708: 'ཨིང་ལིཤ་ཁ',
			1867: 'angla',
			1871: 'inglise',
			1887: 'Yevugbe',
			1908: 'enskt',
			1910: 'انگلیسی',
			1928: 'englanti',
			1953: 'Engeleere',
			1994: 'inglés',
			2016: 'Lungereza',
			2208: 'અંગ્રેજી',
			2315: 'Turanci',
			2326: 'אנגלית',
			2355: 'अंग्रेज़ी',
			2438: 'angol',
			2472: 'enska',
			2534: 'Inggris',
			2569: 'Béarla',
			2595: 'inglese',
			2827: 'tuluttut',
			2971: 'اَنٛگیٖزۍ',
			3018: 'ағылшын тілі',
			3135: 'Gĩthungũ',
			3154: 'Icyongereza',
			3164: 'англисче',
			3276: '영어',
			3589: 'ອັງກິດ',
			3612: 'angļu',
			3718: 'lingɛlɛ́sa',
			3734: 'anglų',
			3814: 'Lingelesa',
			3853: 'Englesch',
			3879: 'англиски',
			3964: 'ഇംഗ്ലീഷ്',
			4003: 'Ingliż',
			4100: 'इंग्रजी',
			4393: 'Anglisy',
			4409: 'Αγγλικά',
			4447: 'англи',
			4517: 'Inggeris',
			4754: 'अङ्ग्रेजी',
			4934: 'eaŋgalsgiella',
			4963: 'isi-Ngisi',
			5024: 'engelsk',
			5043: 'անգլերեն',
			5045: 'ଇଂରାଜୀ',
			5047: 'İngilizce',
			5048: 'ინგლისური',
			5049: 'ಇಂಗ್ಲೀಷ್',
			5050: 'အင်္ဂလိပ်',
			5056: 'Oyibo',
			5073: '英語',
			5277: 'Ingliffa',
			5298: 'англисаг',
			5391: 'ਅੰਗਰੇਜ਼ੀ',
			5580: 'angielski',
			5598: 'inglês',
			5650: 'انګلیسي',
			5676: 'Ingles Simi',
			5778: 'englais',
			5779: 'engleză',
			5807: 'Icongereza',
			5812: 'английский',
			5912: 'Anglëe',
			6014: 'Beurla',
			6142: 'Chirungu',
			6166: 'ꑱꇩꉙ',
			6210: 'ඉංග්‍රීසි',
			6241: 'angleščina',
			6242: 'angličtina',
			6259: 'Ingiriisi',
			6405: 'anglisht',
			6411: 'inglés',
			6412: 'anglès',
			6413: 'anglais',
			6414: 'English',
			17045: 'English',
			6415: 'Englisch',
			6484: 'engelska',
			6589: 'ஆங்கிலம்',
			6696: 'ఆంగ్లం',
			6762: 'อังกฤษ',
			6783: 'དབྱིན་ཇིའི་སྐད།',
			6799: 'እንግሊዝኛ',
			6891: 'lea fakapālangi',
			7019: 'iňlisçe',
			7078: 'ئىنگلىزچە',
			7087: 'англійська',
			7147: 'انگریزی',
			7224: 'Tiếng Anh',
			7395: 'Saesneg',
			7406: 'Ingelsk',
			7742: 'ענגליש',
			7861: '英文',
			7883: 'i-English',
			7947: 'engelsk',
			8013: 'Linglänapük',
		},
		iso: 'eng',
		name: 'English',
		filesets: 113,
		alt_names: [
			'Engels',
			'Borɔfo',
			'እንግሊዝኛ',
			'الإنجليزية',
			'angilɛkan',
			'ingelesa',
			'англійская',
			'ইংরেজি',
			'saozneg',
			'английски',
			'អង់គ្លេស',
			'ингалсан',
			'а҆нглі́йскїй',
			'Sowsnek',
			'engleski',
			'angličtina',
			'engelsk',
			'ཨིང་ལིཤ་ཁ',
			'angla',
			'inglise',
			'Yevugbe',
			'enskt',
			'انگلیسی',
			'englanti',
			'Engeleere',
			'inglés',
			'Lungereza',
			'અંગ્રેજી',
			'Turanci',
			'אנגלית',
			'अंग्रेज़ी',
			'angol',
			'enska',
			'Inggris',
			'Béarla',
			'inglese',
			'tuluttut',
			'اَنٛگیٖزۍ',
			'ағылшын тілі',
			'Gĩthungũ',
			'Icyongereza',
			'англисче',
			'영어',
			'ອັງກິດ',
			'angļu',
			'lingɛlɛ́sa',
			'anglų',
			'Lingelesa',
			'Englesch',
			'англиски',
			'ഇംഗ്ലീഷ്',
			'Ingliż',
			'इंग्रजी',
			'Anglisy',
			'Αγγλικά',
			'англи',
			'Inggeris',
			'अङ्ग्रेजी',
			'eaŋgalsgiella',
			'isi-Ngisi',
			'անգլերեն',
			'ଇଂରାଜୀ',
			'İngilizce',
			'ინგლისური',
			'ಇಂಗ್ಲೀಷ್',
			'အင်္ဂလိပ်',
			'Oyibo',
			'英語',
			'Ingliffa',
			'англисаг',
			'ਅੰਗਰੇਜ਼ੀ',
			'angielski',
			'inglês',
			'انګلیسي',
			'Ingles Simi',
			'englais',
			'engleză',
			'Icongereza',
			'английский',
			'Anglëe',
			'Beurla',
			'Chirungu',
			'ꑱꇩꉙ',
			'ඉංග්‍රීසි',
			'angleščina',
			'Ingiriisi',
			'anglisht',
			'anglès',
			'anglais',
			'English',
			'Englisch',
			'engelska',
			'ஆங்கிலம்',
			'ఆంగ్లం',
			'อังกฤษ',
			'དབྱིན་ཇིའི་སྐད།',
			'lea fakapālangi',
			'iňlisçe',
			'ئىنگلىزچە',
			'англійська',
			'انگریزی',
			'Tiếng Anh',
			'Saesneg',
			'Ingelsk',
			'ענגליש',
			'英文',
			'i-English',
			'Linglänapük',
		],
		autonym: 'English',
		glotto_id: 'stan1293',
		vernacular_name: 'English',
		id: 17045,
		englishName: 'English: USA',
		bibles: 180,
	},
];
