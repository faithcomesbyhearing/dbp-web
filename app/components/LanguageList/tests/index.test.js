import React from 'react';
import { render } from '@testing-library/react';
import { languages } from '../../../utils/testUtils/languageData';
import { LanguageList } from '..';

jest.mock('react-virtualized', () => ({
	List: ({ rowRenderer, rowCount }) => {
		const components = [];
		for (let i = 0; i < rowCount; i++) {
			components.push(rowRenderer({ index: i, style: {}, key: `${i}_row` }));
		}
		return <>{components}</>;
	},
	AutoSizer: ({ children }) => children({ width: 150, height: 50 }),
}));

const props = {
	languages,
	setActiveIsoCode: jest.fn(),
	toggleLanguageList: jest.fn(),
	toggleVersionList: jest.fn(),
	getLanguages: jest.fn(),
	filterText: '',
	active: true,
	loadingLanguages: false,
	languageCode: 17045,
};

describe('LanguageList component', () => {
	it('should match snapshot of active list', () => {
		const { asFragment } = render(<LanguageList {...props} />);
		expect(asFragment()).toMatchSnapshot();
	});
	it('should match snapshot of active list with an applied filter', () => {
		const { asFragment } = render(
			<LanguageList {...props} filterText={'en'} />,
		);
		expect(asFragment()).toMatchSnapshot();
	});
});
