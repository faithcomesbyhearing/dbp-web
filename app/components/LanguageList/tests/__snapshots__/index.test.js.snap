// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LanguageList component should match snapshot of active list 1`] = `
<div
  className="text-selection-section"
>
  <div
    className="language-name-list"
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
  >
    <div
      className="pull-down-refresh"
      style={
        {
          "height": 0,
          "maxHeight": 0,
        }
      }
    >
      <span
        style={
          {
            "textAlign": "center",
            "width": "100%",
          }
        }
      >
        Pull to Refresh
      </span>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Azerbaijani, North"
    >
      <h4
        className=""
      >
        AZƏRBAYCAN
         - ( Azerbaijani, North )
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Aari"
    >
      <h4
        className=""
      >
        Aari
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Abau"
    >
      <h4
        className=""
      >
        Abau
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Abiji"
    >
      <h4
        className=""
      >
        Abiji
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Abun"
    >
      <h4
        className=""
      >
        Abun
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Aceh"
    >
      <h4
        className=""
      >
        Aceh
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Achagua"
    >
      <h4
        className=""
      >
        Achagua
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Acholi"
    >
      <h4
        className=""
      >
        Acholi
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Achuar-Shiwiar"
    >
      <h4
        className=""
      >
        Achuar-Shiwiar
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Aché"
    >
      <h4
        className=""
      >
        Aché
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Achi"
    >
      <h4
        className=""
      >
        Achí
         - ( Achi )
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Achi de Cubulco"
    >
      <h4
        className=""
      >
        Achí, Cubulco
         - ( Achi de Cubulco )
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Adele"
    >
      <h4
        className=""
      >
        Adele
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Adioukrou"
    >
      <h4
        className=""
      >
        Adioukrou
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Adzera"
    >
      <h4
        className=""
      >
        Adzera
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Agta, Dupaninan"
    >
      <h4
        className=""
      >
        Agarabi
         - ( Agta, Dupaninan )
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Agta, Dupaninan"
    >
      <h4
        className=""
      >
        Agta, Dupaninan
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Agta, Pahanan"
    >
      <h4
        className=""
      >
        Agta, Pahanan
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Awakateko"
    >
      <h4
        className=""
      >
        Aguacateco
         - ( Awakateko )
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="English: USA"
    >
      <h4
        className="active-language-name"
      >
        English
         - ( English: USA )
      </h4>
    </div>
  </div>
</div>
`;

exports[`LanguageList component should match snapshot of active list with an applied filter 1`] = `
<div
  className="text-selection-section"
>
  <div
    className="language-name-list"
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
  >
    <div
      className="pull-down-refresh"
      style={
        {
          "height": 0,
          "maxHeight": 0,
        }
      }
    >
      <span
        style={
          {
            "textAlign": "center",
            "width": "100%",
          }
        }
      >
        Pull to Refresh
      </span>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="English: USA"
    >
      <h4
        className="active-language-name"
      >
        English
         - ( English: USA )
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Abun"
    >
      <h4
        className=""
      >
        Abun
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Abau"
    >
      <h4
        className=""
      >
        Abau
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Acholi"
    >
      <h4
        className=""
      >
        Acholi
      </h4>
    </div>
    <div
      className="language-name"
      onClick={[Function]}
      role="button"
      style=""
      tabIndex={0}
      title="Aceh"
    >
      <h4
        className=""
      >
        Aceh
      </h4>
    </div>
  </div>
</div>
`;
